docs/src/tutorials/index.rst

```diff
@@ -19,7 +19,8 @@ A collection of themed practical lessons to work though.
 
     .. grid-item-card::
         :text-align: center
-        :link: region-manifold-extraction.html
+        :link: gv-tutorials-region-manifold-extraction
+        :link-type: ref
```

Comment:
Adopt the best practice of using a tutorial `ref` link within the jupyter notebook, which is overall more robust.